src/Console/ConfigurationResolver.php

```diff
@@ -359,6 +365,19 @@ public function getLinter(): LinterInterface
         return $this->linter;
     }
 
+    public function getParallel(): int
+    {
+        if (null === $this->parallel) {
+            if (null === $this->options['parallel']) {
+                $this->parallel = $this->getConfig()->getParallel();
```

Comment:
so, when running in parallel, by default we gonna have `1` parallel execution?
would be great to autodetect the number

Reply:
Yeah, my idea is to keep the current flow (1 single process) as a default and run it in parallel only if the user specify it

Reply:
while it's interesting to allow to set the amount explicitly, having autodiscovery of it would be awesome too